test/e2e/storage/volume_provisioning.go

```diff
@@ -716,6 +790,37 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 			Expect(claim.Status.Phase).To(Equal(v1.ClaimPending))
 		})
 	})
+
+	framework.KubeDescribe("GlusterDynamicProvisioner", func() {
+		It("should create and delete persistent volumes [fast]", func() {
```

Comment:
is it really fast?

Reply:
Not fast if there is potential for 20m wait.

Reply:
ok. I will mark it as `slow` then.